src/components/DataRecordingGridItem.tsx

```diff
@@ -20,6 +20,9 @@ interface DataRecordingGridItemProps {
   selected: boolean;
   onSelectRow?: () => void;
   startRecording: () => void;
+
+  // For guided tour.
+  cardId?: string;
```

Comment:
Is this used?

Reply:
No, not used. Must have forgotten to remove remnants of a different approach. Removed here:  https://github.com/microbit-foundation/ml-trainer/pull/321/commits/46b2ba673c135d20f9886be106bd2413105f7ddb